django/econsensus/publicweb/views.py

```diff
@@ -805,3 +805,15 @@ def make(cls):
         def search_view(request, *args, **kwargs):
             return cls()(request, *args, **kwargs)
         return login_required(search_view)
+
+class ChangeOwnerView(UpdateView):
+    model = OrganizationOwner
+    form_class = ChangeOwnerForm
+    template_name_suffix = '_update_form'
+    def get_success_url(self):
+        return reverse('organization_admin',
+                kwargs={'organization_pk': self.object.organization.pk})
+    def get_form_kwargs(self):
+        kwargs = super(ChangeOwnerView, self).get_form_kwargs()
+        kwargs.update({'currentOrgPk': self.object.organization.pk})
+        return kwargs
```

Comment:
pep8 isn't the be all and end all, but it is a good start. what editor do you use? we can probably get you set up with an auto checker.